spec/models/state_file_az_intake_spec.rb

```diff
@@ -311,4 +311,20 @@
       end
     end
   end
+
+  describe "invalid_df_w2?" do
+    let(:intake) { create :state_file_az_intake }
+
+    it 'does not validate locality names against known lists' do
+      df_w2 = intake.direct_file_data.w2s[0]
+      df_w2.LocalityNm = "JUNO"
+      expect(intake.invalid_df_w2?(df_w2)).to eq false
+    end
```

Comment:
i can never decide whether it's superfluous to test that something is *not* happening, but my instinct here is that in future, without the context that ny w2s came first, this will look odd